tests/test_cpio.py

```diff
@@ -4,6 +4,7 @@
 import unittest
 import warnings
 
+import xcp.cpiofile
```

Comment:
There's already a line just after this to import from xcp.cpiofile.

Reply:
yes, it's a bit a matter of taste, between importing every used symbol from the module namespace, and explicitly referencing the module in the code.  I did not feel like importing 3 more symbols to use each of them only once, but in the end they're used twice each, and the code will be both a bit more consistent and readable.

Reply:
reworked